index.bs

```diff
@@ -212,6 +212,32 @@ examples that violate the above rules are <code>XMLHttpRequest</code> and
 initialisms, even if they are repeated.
 </div>
 
+<h3 id="prefer-dict-to-bool">Prefer dictionary parameters over boolean parameters or other unreadable parameters</h3>
+
+APIs should generally prefer dictionary parameters over booleans.
```

Comment:
"Over boolean parameters" is helpful since we actually keep using booleans inside the dictionary :)